model/pom.xml

```diff
@@ -103,6 +103,7 @@
       <plugin>
         <groupId>io.smallrye</groupId>
         <artifactId>smallrye-open-api-maven-plugin</artifactId>
+        <version>2.1.9</version>
```

Comment:
better define the version in `pluginManagement`

Reply:
ah you're right, I keep forgetting about this